lib/utils/assign.js

```diff
@@ -0,0 +1,3 @@
+// Aliasing Object.assign allows the minifier to shorten `Object.assign` to a single character variable.
+// Shortens 13 bytes to 1 at the cost of another allocation.
+export default Object.assign
```

Comment:
I kinda thought this affords that opportunity. I also just got tired of so many `Object.assign`'s.